script.js

```diff
@@ -0,0 +1,74 @@
+let bookArray = [];
+
+function bookObj(bookTitle, authorName) {
+  const eachBook = {
+    title: bookTitle,
+    author: authorName,
+  };
+
+  bookArray.push(eachBook);
+}
+
+function addToLocalStorage() {
+  const stringifyArray = JSON.stringify(bookArray);
+  localStorage.setItem('storedBooks', stringifyArray);
+}
+
+function booksFilter(index) {
+  bookArray = bookArray.filter((book) => book !== bookArray[index]);
+}
```

Comment:
Great job team 🤝

- [ ] [Optional] You may write all your functions and methods using ES6 syntax as this is the recommended way lately.

Reply:
thank you